package.json

```diff
@@ -46,7 +48,7 @@
     "build/contracts/Migrations.json",
     "build/contracts/TokenConservation.json",
     "build/types",
-    "typescript/common",
+    "lib/common",
```

Comment:
Kind of unrelated to this PR, but I only noticed this how - shouldn't the `esm` directory also be included in the package so that `jsnext:main` and `module` paths are valid?

Reply:
Yes will add that.